client/src/unifycr-dirops.c

```diff
@@ -106,7 +106,7 @@ DIR *UNIFYCR_WRAP(opendir)(const char *name)
          */
         meta->size = sb->st_size;
         meta->chunks = sb->st_blocks;
-        meta->real_size = sb->st_size;
+        meta->size = sb->st_size;
```

Comment:
This meant to be log_size instead?

Reply:
@sandrain getting ready to merge this, but wanted to check-in about this comment first. Line 107 & 109 are the same so we were assuming you meant to have line 109 be meta->log_size?